interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -925,17 +922,15 @@ def test_listFromFile(self):
 
     def test_listFromCti(self):
         gas = ct.Solution("h2o2.xml", transport_model=None)
-        p = os.path.dirname(__file__)
-        with open(pjoin(p, '..', 'data', 'h2o2.cti')) as f:
+        with open(self.cantera_data_path / "h2o2.cti") as f:
             R = ct.Reaction.listFromCti(f.read())
```

Comment:
I think you can use `Path.read_text()` here, if I'm interpreting the documentation of `read()` correctly.

Reply:
This occurs in several places.